### ISSUE TITLE
Audio Codec Options - AAC / Dolby

### ISSUE THREAD

**Issue by rcottingham** [cta-wave/Test-Content#58]
Here are the final "sparse matrix" files used for AAC and Dolby codecs. The latest uploads and generated tests adhere to these requirements.

[CTA WAVE_Dolby_AC4_E-AC-3_Matix_20230807.xlsx](https://github.com/cta-wave/Test-Content/files/13719003/CTA.WAVE_Dolby_AC4_E-AC-3_Matix_20230807.xlsx)
[CTA WAVE_AAC_Matix_20230807.xlsx](https://github.com/cta-wave/Test-Content/files/13719005/CTA.WAVE_AAC_Matix_20230807.xlsx)

Apologies but I couldn't find the location of the similar files used for AVC etc, so posting here for now.

**Comment by rcottingham** [comment]
Updated AAC matrix to include at17 - Long Duration.

[CTA.WAVE_AAC_Matix_20250203.xlsx](https://github.com/user-attachments/files/18643464/CTA.WAVE_AAC_Matix_20250203.xlsx)

**Comment by rbouqueau** [comment]
Can this be closed?

**Comment by rcottingham** [comment]
Yes - audio options have been updated in the latest published DPCTF spec and here: https://cta-wave.github.io/Test-Content/

**Comment by rbouqueau** [comment]
Thank you. Closing then.
